src/Mobs/Monster.cpp

```diff
@@ -605,6 +605,15 @@ void cMonster::KilledBy(TakeDamageInfo & a_TDI)
 	{
 		m_World->BroadcastSoundEffect(m_SoundDeath, GetPosition(), 1.0f, 0.8f);
 	}
+
+	if (IsTame())
+	{
+		if ((m_MobType == mtWolf) || (m_MobType == mtOcelot) || (m_MobType == mtCat) || (m_MobType == mtParrot))
+		{
+			BroadcastDeathMessage(a_TDI);
+		}
+	}
```

Comment:
This appears to give a death message whenever a tame animal is killed, rather than just when they have a custom name. Is this intentional?

Reply:
Yes, according to the vanilla Minecraft.

Reply:
OK, that's fine then.